assignments/objects.js

```diff
@@ -19,27 +19,77 @@ const example = {
 
 // Write your intern objects here:
 
+const Mitzi = {
+  "id":1,
+  "name":"Mitzi",
+  "email":"mmelloy0@psu.edu",
+  "gender":"F",
+}
+
+const Kennan = {
+  "id":2,
+  "name":"Kennan",
+  "email":"kdiben1@tinypic.com",
+  "gender":"M",
+}
+
+const Kevan = {
+  "id":3,
+  "name":"Kevan",
+  "email":"kummery2@wikimedia.org",
+  "gender":"M",
+}
+
+const Gannie = {
+  "id":4,
+  "name":"Gannie",
+  "email":"gmartinson3@illinois.edu",
+  "gender":"M",
+}
+
+const Antonietta = {
+  "id":5,
+  "name":"Antonietta",
+  "email":"adaine5@samsung.com",
+  "gender":"F",
+}
+
 
 // ==== Challenge 2: Reading Object Data ==== 
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
+console.log(Mitzi.name);
 
 // Kennan's ID
+console.log(Kennan.id);
 
 // Keven's email
+console.log(Kevan.email);
 
 // Gannie's name
+console.log(Gannie.email);
 
 // Antonietta's Gender
+console.log(Antonietta.gender);
 
 // ==== Challenge 3: Object Methods ==== 
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
 // console.log(kennan.speak());
 
+Kennan.speak = function(){
+  return "Hello, my name is Kennan!";
+}
+console.log(Kennan.speak());
```

Comment:
You could make this more flexible by using the `this` keyword:
```js
kennan.speak = function() {
  console.log(`Hello, my name is ${this.name}`);
}
```
The `this` keyword here refers to the scope it's being executed in. Since the speak function is defined within the scope of kennan, it refers to `kennan.name`.